pydantic/main.py

```diff
@@ -835,6 +835,7 @@ class MyModel(BaseModel, extra='allow'):
     def __iter__(self) -> TupleGenerator:
         """So `dict(model)` works."""
         yield from self.__dict__.items()
+        yield from (self.__pydantic_extra__ or {}).items()
```

Comment:
```suggestion
        extra = self.__pydantic_extra__
        if extra:
            yield from extra.items()
```

ready for the :seal: when we drop 3.7.

```
               ___
            .-9 9 `\
          =(:(::)=  ;
            ||||     \
            ||||      `-.
           ,\|\|         `,
          /                \
         ;                  `'---.,
         |                         `\
         ;                     /     |
         \                    |      /
          )           \  __,.--\    /
       .-' \,..._\     \`   .-'  .-'
      `-=``      `:    |   /-/-/`
                   `.__/
```